docs/tutorial/t-spark-service-accounts.md

```diff
@@ -73,25 +70,27 @@ options:
                         Namespace of service account name to use other than primary.
   --primary             Boolean to mark the service account as primary.
 ```
-Service account is an abstraction for a set of associated kubernetes resources needed to run a Spark job. The user can choose to associate configuration properties 
-with the service account that can serve as default while submitting jobs against that service account from any machine within the kubernetes cluster. A typical use 
-of this feature would look like this.
+Service account is an abstraction for a set of Kubernetes resources needed to run a Spark job. The user can choose to associate configuration properties 
+with service accounts that will serve as default for related job submissions across the cluster. A typical use of this feature would look like this.
 
 ```bash
-spark-client.service-account-registry --username demouser --namespace demonamespace create --properties-file /home/demouser/conf/spark-defaults.conf --conf spark.app.name=demo-spark-app --conf spark.executor.instances=3
+spark-client.service-account-registry create --username demouser --namespace demonamespace  --properties-file /home/demouser/conf/spark-defaults.conf --conf spark.app.name=demo-spark-app --conf spark.executor.instances=3
 ```
 
-The above command sets up a service account for user ```demonamespace:demouser``` for Spark job submission using configuration properties coming from the specified 
-properties file while overriding the configuration properties ```spark.app.name``` and ```spark.executor.instances```.
-
-For [job submission](https://discourse.charmhub.io/t/spark-client-snap-tutorial-spark-submit/8953), this service account along with it's default configuration properties can be used to submit a Spark job. 
+The above command sets up a service account for user ```demonamespace:demouser``` for Spark job submission. Configuration properties are taken from the specified 
+properties file, potentially overriden by configuration properties ```spark.app.name``` and ```spark.executor.instances```.
 
-For example, assuming the properties file provided has configuration details to access data in S3, one could submit a job like
-```bash
-spark-client.submit  --username demouser --namespace demonamespace --deploy-mode cluster --conf spark.app.name=demo-spark-s3-app $S3_PATH_FOR_CODE_FILE
+**_Note:_** The command described above does not create a kubernetes namespace but needs it to be there. It does however create the requested username in the specified and existing namespace. The kubernetes namespace could be created such as:
```

Comment:
perhaps we should expand the functionality of the tool over time to allow for namespace creation?

Reply:
Yes, indeed. I have created an associated ticket ([DPE-2141](https://warthogs.atlassian.net/browse/DPE-2141)) about this